### DISCUSSION TITLE
tsml_defaults include "region"

### DISCUSSION THREAD

**Discussion by jerrytunin** [code4recovery/12-step-meeting-list#1214] [Ideas]
Add the option to set the default region in functions.php.

**Comment by kiyote33** [comment]
Thanks for the suggestion @jerrytunin. I will get this added to our agenda for our next developer meeting (Saturday, Sept 23, 2023) where we discuss the pros & cons of suggestions like this.

**Reply by jerrytunin** [reply]
Excellent, thank you.  This is the scenario:

Default page/homepage is 12-step-meeting-list utilizing "add_action('pre_get_posts', 'tsml_front_page');
" in functions.php

The website is targeted primarily towards 1 particular region but also lists meetings for all regions in the state as well.  We would like to have it default to the primary region and then if someone wants to also see meetings from other regions they can do so from the drop down.

**Comment by joshreisner** [comment]
@jerrytunin typically when people ask this they want to create 'landing pages' for regions that they can link to from the site navigation -- you can do this now using URLs. for an example, see the meetings dropdown on the area72aa.org website

everywhere > https://area72aa.org/meetings/
seattle > https://area72aa.org/meetings/?region=seattle
tacoma > https://area72aa.org/meetings/?region=tacoma

is this similar to your use-case, or are you looking for different functionality?

**Reply by jerrytunin** [reply]
Sort of, but instead of the main meetings page being everywhere we would have it default to the particular region that the site is focused on.

This is the scenario:

Default page/homepage is 12-step-meeting-list utilizing "add_action('pre_get_posts', 'tsml_front_page');
" in functions.php

The website is targeted primarily towards 1 particular region but also lists meetings for all regions in the state as well.  We would like to have it default to the primary and then if someone wants to also see meetings from other regions they can do so from the drop down.
